specfile/specfile.py

```diff
@@ -63,6 +82,8 @@ def __init__(
 
         Args:
             path: Path to the spec file.
+            file: File object representing the spec file.
+            content: String containing the spec file content.
```

Comment:
Can we document the fact that these arguments are mutually exclusive? Probably in the description above, to avoid repetition.

Reply:
Also, maybe it would make sense to change the order and swap `file` and `content`, `file` is on a lower level than `path` and `content` and we can't move `path` for backward compatibility.